zhmccli/_cmd_nic.py

```diff
@@ -148,6 +159,17 @@ def nic_create(cmd_ctx, cpc, partition, **options):
               'Only for OSA and HiperSocket adapters.')
 @click.option('--device-number', type=str, required=False,
               help='The new device number to be used for the NIC.')
+@click.option('--ssc-management-nic', type=bool, required=False,
+              help='Indicates that this NIC should be used as a management '
+              'NIC for Secure Service Container to access the web interface.')
+@click.option('--ssc-ip-address-type', type=str, required=False,
+              help='"ipv4, ipv6, linklocal or dhcp.')
+@click.option('--ssc-ip-address', type=str, required=False,
+              help='IP Address of the SSC management web interface.')
+@click.option('--ssc-mask-prefix', type=str, required=False,
+              help='Network Mask of the SSC management NIC.')
+@click.option('--vlan-id', type=str, required=False,
+              help='VLAN ID of the SSC management NIC ')
```

Comment:
Add to help text: "Only applicable to NICs of ssc type partitions."
TODO: On this one, we need to clarify how flat networking is set.